### ISSUE TITLE
Error when trying to build App. Could be related to redux versions?

### ISSUE THREAD

**Issue by BoweFrankema** [betrueagency/nx-reactjs-elementor#4]
Thanks for building this starter! When trying to get this working I ran into an error during the step

```
nx serve my-project
```

Here is the exact error:

```
TS2345: Argument of type 'EnhancedStore<{ falseEntity: string; }, AnyAction, MiddlewareArray<[ThunkMiddleware<{ falseEntity: string; }, AnyAction, undefined>]>>' is not assignable to parameter of type 'EnhancedStore<any, AnyAction, Middlewares<any>>'.
```

I've tried updating the Package.json to recent versions of redux/redux-toolkit/react-redux but this did not help. I found something here related to the issue but did not got anywhere: https://github.com/reduxjs/redux-thunk/issues/333

Cheers,
Bowe

**Comment by alizarion** [comment]
Hi @BoweFrankema thanks for opening this issue , Indeed there is a dependency management problem, I'm checking that

**Comment by alizarion** [comment]
@BoweFrankema resolved on v1.0.1, don't hesitate to open a new Issue if you encounter another problem 👍
